src/main/kotlin/org/jitsi/nlj/transform/node/incoming/VideoBitrateCalculator.kt

```diff
@@ -60,7 +60,7 @@ class VideoBitrateCalculator(
         when (event) {
             is SetMediaStreamTracksEvent -> {
                 mediaStreamTrackDescs = event.mediaStreamTrackDescs.copyOf()
-                logger.cdebug { "Video bitrate calculator got media stream tracks:\n$mediaStreamTrackDescs" }
+                logger.cdebug { "Setting new media stream tracks with size ${mediaStreamTrackDescs.size}" }
```

Comment:
Why not print the track descriptions?

Reply:
I thought that would make the message too long. I can bring it back.

Reply:
Ok, can't remember if it's printed elsewhere or not.